server/src/main/java/io/kroki/server/Server.java

```diff
@@ -72,7 +79,59 @@ static void start(Vertx vertx, VertxOptions vertxOptions, JsonObject config, Han
     HttpServer server = vertx.createHttpServer(serverOptions);
     Router router = Router.router(vertx);
     BodyHandler bodyHandler = BodyHandler.create(false).setBodyLimit(config.getLong("KROKI_BODY_LIMIT", BodyHandler.DEFAULT_BODY_LIMIT));
-
+    HttpClient delegatorHttpClient = vertx.httpClientBuilder().withRedirectHandler(resp -> {
```

Comment:
according to the document we need to `setFollowRedirects`:

>  The redirect handler is called when a {@code 3xx} response is received and the request is configured to follow redirects with {@link HttpClientRequest#setFollowRedirects(boolean)}.

Reply:
I wrote the reason at [here](https://github.com/yuzutech/kroki/issues/1887#issuecomment-2867744573), but it should be added in comment as you said.

Reply:
As far as I understand, even when using a custom redirect handler we need to declare `setFollowRedirects (true)` otherwise the redirect handler won't be called.

Reply:
https://vertx.io/docs/vertx-web-client/java/#_handling_30x_redirections

> By default the client follows redirections

Also, I added test and it actually works fine without `setFollowRedirects (true)`.